test/integration/setup.ts

```diff
@@ -21,14 +21,14 @@ const copyFolder = (origin: string, destiny: string): void => {
 }
 
 export const createSandboxProject = (sandboxPath: string): string => {
-  rmdirSync(sandboxPath, { recursive: true })
+  rmSync(sandboxPath, { recursive: true, force: true })
```

Comment:
Why are you using force here now?

Reply:
I'm forcing the call to avoid failing if the directory doesn't exist. If you don't face it the call was failing. @Moneyba